Cargo.toml

```diff
@@ -24,7 +24,7 @@ latest_rust = []  # use latest Rust features (otherwise, support Rust 1.34)
 [dependencies]
 base64 = "0.10"
 bincode = "1.0"
-bitcoin = { version = "0.23", features = ["use-serde"] }
+bitcoin = { path = "../rust-bitcoin", features = ["use-serde"] }
```

Comment:
I think we'll want rust-bitcoin to be a fork in the ZenGo-X account, and to be used within a specific branch.